src/integrations/contact-loaders/helpers.js

```diff
@@ -43,11 +43,9 @@ export const finalizeContactLoad = async (
       index * chunkSize,
       (index + 1) * chunkSize
     );
-    await CampaignContact.save(savePortion).catch(err => {
-      // eslint-disable-next-line no-console
-      console.error("Error saving campaign contacts:", campaignId, err);
-      // call the error thing
-    });
+
+    console.log(`Working on importing chunk No: ${index}`);
+    await r.knex.batchInsert("campaign_contact", savePortion);
```

Comment:
I very much support switching to knex commands, but can we preserve the `catch` -- wrapping this in a try...catch and reporting the error.  Bonus points for updating `campaign_admin` fields on failure.